docs/modules/ROOT/pages/usage.adoc

```diff
@@ -337,8 +338,84 @@ Peak Memory: 0B
 2.67 [15 rows, 8.08KB] [5 rows/s, 3.02KB/s]
 ----
 
-=== Tips
+== Configuration & Environment Overrides
+
+The cluster definition also supports overriding configuration properties and environment variables, either per role or per role group, where the more specific override (role group) has precedence over the less specific one (role).
+
+IMPORTANT: Do not override port numbers. This will lead to faulty installations.
+
+=== Configuration Properties
+
+For a role or role group, at the same level of `config`, you can specify: `configOverrides` for:
+
+- `config.properties`
+- `node.properties`
+- `log.properties`
+- `password-authenticator.properties`
+
+For a list of possible configuration properties see: https://trino.io/docs/current/admin/properties.html
+
+[source,yaml]
+----
+workers:
+  roleGroups:
+    default:
+      config: {}
+      replicas: 1
+      configOverrides:
+        config.properties:
+          query.max-memory-per-node: "2GB"
+----
+
+Just as for the `config`, it is possible to specify this at role level as well:
+
+[source,yaml]
+----
+workers:
+  configOverrides:
+    config.properties:
+      query.max-memory-per-node: "2GB"
+  roleGroups:
+    default:
+      config: {}
+      replicas: 1
+----
 
-If you work with opa, try changing some RegoRule entries to false and see if you are not allowed to e.g. list tables or schemas.
+All override property values must be strings. The properties will be passed on without any escaping or formatting.
+
+=== Environment Variables
+
+Environment variables can be (over)written by adding the `envOverrides` property.
+
+For example per role group:
+
+[source,yaml]
+----
+workers:
+  roleGroups:
+    default:
+      config: {}
+      replicas: 1
+      envOverrides:
+        MY_ENV_VAR: "MY_VALUE"
+----
+
+or per role:
+
+[source,yaml]
+----
+workers:
+  envOverrides:
+    MY_ENV_VAR: "MY_VALUE"
+  roleGroups:
+    default:
+      config: {}
+      replicas: 1
+----
+
+Here too, overriding properties such as `http-server.https.port` will lead to broken installations.
+
+=== Tips
```

Comment:
Is this "Tips" at the right level of indentation? Feels like it doesn't belong into the override section (i.e. should be one level higher)

Reply:
good point yeah